app/(auth)/signup/page.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from 'next/link';
+
+import SignUpForm from './_components/signupForm';
+
+export default function SignUp() {
+  return (
+    <div className="flex flex-col items-center">
+      <SignUpForm />
+      <p className="mt-5 text-black">
+        이미 가입하셨나요?{' '}
+        <span className="text-[#5534DA] underline">
+          <Link href="/signin">로그인하기</Link>
+        </span>
```

Comment:
이렇게 표현해줘도 괜찮을 것 같아요!
html을 작성할때는 가능하다면 태그를 적게 사용해주는게 좋아요!
```suggestion
        <Link href="/signin" className="text-[#5534DA] underline">
          로그인하기
        </Link>
```